web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -1,12 +1,48 @@
 # PDEP-10: PyArrow as a required dependency for default string inference implementation
 
-- Created: 17 April 2023
-- Status: Accepted
+- Created: 17 April 2023 (updated May 8, 2024)
+- Status: Rejected
 - Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
               [#52509](https://github.com/pandas-dev/pandas/issues/52509)
 - Author: [Matthew Roeschke](https://github.com/mroeschke)
           [Patrick Hoefler](https://github.com/phofl)
-- Revision: 1
+- Revision: 2
+
+# Note
+
+This PDEP was originally accepted on May 8, 2023. However, after reviewing feedback posted
+on the feedback issue [#54466](https://github.com/pandas-dev/pandas/issues/54466), we, the members of
+the core team, have not decided with moving forward with this PDEP for pandas 3.0.
```

Comment:
Remove remainder of this text from pdep-10, and move it to pdep-15.  Add a reference to PDEP-15 here.